articles/aks/web-app-routing.md

```diff
@@ -69,6 +69,14 @@ Import the SSL certificate into Azure Key Vault.
 az keyvault certificate import --vault-name <KeyVaultName> -n <KeyVaultCertificateName> -f aks-ingress-tls.pfx
 ```
 
+### Import certificate to Azure Key Vault with password if specified while exporting the SSL cert 
+
+Import the SSL certificate into Azure Key Vault with password
+
+```azurecli-interactive
+az keyvault certificate import --vault-name <KeyVaultName> -n <KeyVaultCertificateName> -f aks-ingress-tls.pfx --password <Password>
```

Comment:
I suggest to update to 
`az keyvault certificate import --vault-name <KeyVaultName> -n <KeyVaultCertificateName> -f aks-ingress-tls.pfx [--password <certificate password if specified>]`